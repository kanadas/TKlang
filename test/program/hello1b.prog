stream main
input Input
state
output print = 'h'
initial print = 'a'
