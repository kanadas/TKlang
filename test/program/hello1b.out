program/hello1b.prog
hh